PatrickNiyogitare28/Fimbo-Nodejs-Server
# Fit: First commit adding Server application features

Thank you for your contribution to the Fimbo Nodejs Server. 

### Before submitting this PR, please make sure:

 
### Testing 
- You should have Nodejs installed
- Use Postiman API Tester
- Install and Configure Mysql Server
- Configure connnection to Mysql server in `server.js`
